src/probabilistic/CardinalityCounter.cc

```diff
@@ -134,6 +134,10 @@ void CardinalityCounter::AddElement(uint64_t hash) {
  **/
 double CardinalityCounter::Size() const {
     double answer = 0;
+
+    // Avoid a divide-by-zero here.
+    ASSERT(m > 0);
```

Comment:
The bif doc states that it returns `-1.0` if empty, so maybe that'd be okay to return here instead of ASSERT which may not produce any code?